src/cpu/intel.rs

```diff
@@ -41,6 +41,21 @@ impl Feature {
     }
 }
 
+pub(crate) const ADX: Feature = Feature {
```

Comment:
You can constants are the same as those used in the BoringSSL functions that were previously being called in curve25519.c.